openfile-perms.yml

```diff
@@ -1,7 +1,8 @@
+# corollary to mkdir-perms
```

Comment:
Must this "one eponymous rule id per file" idiom be followed? Personally, I'd prefer to put these in one file like `osfile-perms.yml`.

Reply:
SGTM.  And yes, multiple rules can be in a single file.